docs/reference.md

```diff
@@ -149,6 +149,14 @@ Create a new server action.
 hemera.subscribe(pattern, cb)
 ```
 
+### Hemera.add()
+
+The topic is subscribed on NATS and can be called from any client.
```

Comment:
I would describe what the method does.
e.g "Register a new server action which can be called from any client."